extensions/amp-bind/0.1/bind-expr-impl.js

```diff
@@ -884,4 +884,20 @@ Parser.prototype = parser;parser.Parser = Parser;
 return new Parser;
 })();
 
-exports.parser = parser;
+
+if (typeof require !== 'undefined' && typeof exports !== 'undefined') {
```

Comment:
It's part of the autogenerated file. I'm not sure if they added this in the newest version of jison or if @choumx just removed it from previous versions.